quickwit-metastore/src/tests.rs

```diff
@@ -297,6 +227,16 @@ pub mod test_suite {
                 .await
                 .unwrap();
             assert!(matches!(result, ()));
+            assert_eq!(
+                metastore.list_all_splits(index_id).await.unwrap(),
+                vec![SplitMetadataAndFooterOffsets {
```

Comment:
Maybe a helper function can help us build this vector and reduce the verbosity of these tests since this pattern is repeated quite a few times.